src/Components/WeatherDetails/WeatherDetails.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { formatNumberToCelcius } from '../Utils/Utils';
```

Comment:
rather use index.js inside utils directory to avoid such a strange path.
You will have just ../Utils

and one more thing - utils directory should be placed in COMMON directory not in components directory